spacy_llm/tests/tasks/test_ner.py

```diff
@@ -949,3 +949,20 @@ def test_regression_span_task_response_parse(
     doc = docs[0]
     pred_ents = [(ent.text, ent.label_) for ent in doc.ents]
     assert pred_ents == gold_ents
+
+
+@pytest.mark.external
+@pytest.mark.skipif(has_openai_key is False, reason="OpenAI API key not available")
+def test_entity_with_comma(fewshot_cfg_string_v3_lds):
+    config = Config().from_str(fewshot_cfg_string_v3_lds)
+    nlp = spacy.util.load_model_from_config(config, auto_fill=True)
+    text = "Somebody with the name 'Louis, the XVIIth', and Bob both live in Ireland."
+    doc = nlp(text)
+    ents = doc.ents
+    assert len(ents) == 3
+    assert ents[0].text == "'Louis, the XVIIth'" or "Louis, the XVIIth"
```

Comment:
While running this locally, some runs showed the prediction with quotes, others without. Both are reasonable and fine for the purpose of this test, IMO.